@xen-orchestra/web/src/pages/vm/new.vue

```diff
@@ -6,6 +6,12 @@
         <VtsSelect :id="poolSelectId" accent="brand" />
       </template>
     </UiHeadBar>
+    <UiAlert v-if="vmState.pool" accent="info">
```

Comment:
Add the class to add some padding:
```suggestion
    <UiAlert v-if="vmState.pool" accent="info" class="card-container">
```